### DISCUSSION TITLE
Open documents in client applications

### DISCUSSION THREAD

**Discussion by MelanieO365** [microsoft-search/pnp-modern-search#3644] [General]
**Version used**
4.9.3.0

**Describe the bug**
Configurated that documents should open in client applications but when using PnP Modern Search, the result layout "details list" is not opening the documents in the client application.

**To Reproduce**
1. activate the site collection feature to open the documents by default in the client application
2. configure the search results webpart to show the layout as "details list"
3. Test by clicking on the title field to open the document. It will open the document in the browser

**Expected behavior**
Expected the documents to open in the client application. Maybe add a toggle to configure the results webpart to enable "open documents in client application"

**Desktop (please complete the following information):**
 - Browser Chrome

**Comment by kasperbolarsen** [comment]
If you use the Handlebar option you can tweak the URLs to either open the document in the browser or in the application.
like `<a href="{{Path}}?web=1">link</a>`
web=0 is "open in client", and web=1 is open in browser :-)

**Reply by MelanieO365** [reply]
I tested already "web=0" but that is resulting in downloading the file. That is not the result the customer wants.

**Reply by MelanieO365** [reply]
I have also tested the following but this does not work either: 

`<div>
        <span>
               <a href="{{#if (eq (slot item @root.slots.FileType) "docx") }} ms-word:ofe|u|{{else if (eq (slot item @root.slots.FileType) "xlsx") }}ms-excel:ofe|u|{{else if (eq (slot item @root.slots.FileType) "pptx") }}
		ms-powerpoint:ofe|u|{{else}}{{/if}}{{slot item @root.slots.PreviewUrl}}" target="_blank" style="color: {{@root.theme.semanticColors.link}}">{{Title}}</a>
        </span>
    </div>`

**Reply by kasperbolarsen** [reply]
web=0 should open the file in the client, and I have tested it with success on my laptop. Is it possible that you have a local issue on your computer?
And just for your info: in most cases we have to ask people to upgrade to the current version as we really can't support older versions

**Reply by MelanieO365** [reply]
I did an update to the latest version. In Edge it is working but not in Chrome.
